NEWS.md

```diff
@@ -84,6 +84,12 @@
 
 * Increase the default `nbin` of `guide_colourbar()` to place the ticks more precisely (#3508, @yutannihilation).
 
+* Modified `manual_scale` to match `values` with the order of `breaks` whenever
+  `values` is an unnamed vector. Previously, unnamed `values` would be matched
+  to the limits of the scale and ignore the order of any `breaks` provided. Note
+  that this may change the appearance of plots that previously relied on the
+  unordered behaviour. (#2429, @idno0001)
```

Comment:
Please move the period to the very end of the line: "unordered behaviour (#2429, @idno0001)."